Makefile

```
@@ -24,6 +24,17 @@ chromedp.run:
 
 .PHONY: release
 
+test-docker:
```

Could use dots here for consistency

```suggestion
test.docker:
```